chart/values.yaml

```diff
@@ -43,6 +43,12 @@ scrapeConfigUpdateInterval: 10m
 # components and the tested remoteStorages.
 writeConcurrency: 16
 
+# replicas defines how many pods of writers to deploy.
+# each replica will scrape targetsCount targets and will have
+# its own extra label `replica` attached to written time series.
+# this option is useful for scaling the writers horizontally.
+writeReplicas: 1
```

Comment:
This setting must be optional. It must be set to 1 if not explicitly specified in the config

Reply:
I added default value in chart/templates/vmagent/deployment.yaml

Reply:
made param optional